main.go

```diff
@@ -92,3 +99,80 @@ func retrieveComments(segment string) *string {
 	}
 	return completion
 }
+
+func EvaluateReviewQuality(pr *github.PullRequest, client github_client.GithubRepoClient) error {
```

Comment:
Missing Test: There are no unit tests for the EvaluateReviewQuality() function. Please add some tests for different scenarios to ensure it works as intended.

Reply:
Thank you for your feedback! The AI gives your comment a score of 4. The comment is practical and actionable, but could be more specific on what scenarios need to be tested.

Reply:
ya, this function is a bit of a beast. should probably be abstracted to it's own file as well as tested. [I'll backlog it.](https://github.com/birdsean/review-droid/issues/6)